README.md

```diff
@@ -1,2 +1,44 @@
-# verifact_backend
-VeriFact back end
+# Getting Started
+
+## Install Django
+
+### MacOS
```

Comment:
for future might be easier for users/devs to group by OS then step, i.e. 
## MacOS:
### Step 1
### Step 2

## Windows:
### Step 1
### Step 2

if that makes sense to you I'll make that change along with adding Windows instructions on my next PR

Reply:
I think that makes sense. I prefer that the PR creator makes changes based on suggestions as it's their responsibility that the code is in good shape. So I'll make that change.

Reply:
Updated